### ISSUE TITLE
Renaming Github URL causes update issues to Packagist

### ISSUE THREAD

**Issue by jaypatel512** [composer/packagist#451]
I just renamed https://github.com/paypal/rest-api-sdk-php to https://github.com/paypal/PayPal-PHP-SDK

The package name is packagist is the same for now. We want to change its name, but we could do it by having both at the same time, and deprecating the old one later. 

However, the issue right now is, that the users are getting error :+1: 
```
Loading composer repositories with package information
Updating dependencies (including require-dev)

Installing paypal/rest-api-sdk-php (v0.13.2) Downloading: connection... Failed to download paypal/rest-api-sdk-php from dist: The "https://api.github.com/repos/paypal/rest-api-sdk-php/zipball/784100f89b9ea13640bc207d57561bad9e92554e" file could not be downloaded (HTTP/1.1 404 Not Found) Now trying to download from source
Installing paypal/rest-api-sdk-php (v0.13.2)
Cloning 784100f89b9ea13640bc207d57561bad9e92554e

[RuntimeException]
Failed to clone https://c2d129dea09c83be7e7125035f76a25eab9d0bb9:***@github
.com/paypal/rest-api-sdk-php.git, git was not found, check that it is insta
lled and in your PATH env.
sh: git: command not found

update [--prefer-source] [--prefer-dist] [--dry-run] [--dev] [--no-dev] [--lock] [--no-plugins] [--no-custom-installers] [--no-scripts] [--no-progress] [--with-dependencies] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [--ignore-platform-reqs] [packages1] ... [packagesN]
```

Updating the canonical URL seem to have not fixed the issue.

**Comment by Seldaek** [comment]
Actually now updating the URL force a complete update of all existing tags to make sure URLs get updated.

**Comment by addshore** [comment]
I hate to poke such an old ticket but we seem to have run into the same issue with out wikibase/javascript-api package.
Yesterday we update the git URL from http://github.com/wikimedia/mediawiki-extensions-WikibaseJavaScriptApi to http://github.com/wikimedia/wikibase-javascript-api
The packagist UI looks fine but none of the tags seem to have been updated to use the new URL.

I also created a test branch and tag https://github.com/wikimedia/wikibase-javascript-api/releases/tag/2.2.2-addshore, and packagist seemed to update with the new branch but not the new tag when hitting the update button.

> Actually now updating the URL force a complete update of all existing tags to make sure URLs get updated.

Is it possible this has broken at some point in the last 3.5 years?

The ticket on our issue tracker is https://phabricator.wikimedia.org/T203162
